### PR TITLE
Add stubs for core and collections

### LINKED ISSUE #13: Add stub page for collections.html
Let's orphan collections.html and add a statement to redirect to main landing page.

### PR DESCRIPTION
**Pull request by oraNod**
Fixes #13

### CODE REVIEW

**Comment by samccann** [comment]
I don't follow what this is doing.
Core still needs links to the core docset. docs.ansible.com/ansible-core.  We could link to that directly from the top nav where you have core now.

File: templates/collections.html [hunk 1]
```diff
@@ -1,8 +1,20 @@
...
+    </p>
+    <p>
+      Here are some links you might find useful instead:
+      <ul>
+        <li>
+          <a href="{{ pages.site.index }}">Ansible community documentation home page</a>
+        </li>
+        <li>
+          <a href="{{ links.pkg_docs.index }}">Ansible core documentation</a>
+        </li>
```
Comment (samccann) [inline templates/collections.html:16]: Seems this one should link to the collections index, not the main core or community docs. But couldn't we do this with an http redirect in ansible/docsite instead?

File: templates/core.html [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
+      Here are some links you might find useful instead:
+      <ul>
+        <li>
+          <a href="{{ pages.site.index }}">Ansible community documentation home page</a>
+        </li>
+        <li>
+          <a href="{{ links.core_docs.collections_index }}">Indexes of all modules and plugins</a>
+        </li>
+      </ul>
     </p>
```
Comment (samccann) [inline templates/core.html:18]: This one should link to core docs. That said, I'm not sure I agree with stubbing it out. The current core page has the core link as well as the japanese  translation link. We'll always need homes for those, don't you think>?

Comment (samccann) [reply templates/core.html:18]: ok realized on another PR  that collections and core are now on the ecosystem page. So these stubs make more sense to me now. I'd say:
1 - make sure the links are correct per my comments above
2 - consider making the core in the top nav go directly to core devel

File: templates/collections.html (hunk 1 above)
Comment (oraNod) [reply templates/collections.html:16]: doh. fixed the link.

we could add a redirect but a stub page might get us more feedback after we go live. I'm going to add feedback links in a separate PR.

File: templates/core.html (hunk 2 above)
Comment (oraNod) [reply templates/core.html:18]: 1. done. 2. good call.

Comment (oraNod) [reply templates/core.html:18]: @samccann I'm of two minds about the stub page for core.html now. I've pushed changes that link directly from the nav bar to the core docs. However that goes against the pattern of nav link to docsite page with the other nav items.

Maybe we can go ahead with this PR as-is and revisit core.html when we incorporate the translated docs?

Let me know what you think.

Comment (samccann) [reply templates/core.html:18]: sounds like a plan stan

**Comment by samccann** [comment]
I can't build this locally. I could for the footer PR so I think my local reqs are correct. This is the error I get:
`Traceback (most recent call last):
  File "/home/samccann/venvs/jinja-doc-devel/lib64/python3.11/site-packages/staticjinja/staticjinja.py", line 408, in render_template
    rule = self.get_rule(template.name)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samccann/venvs/jinja-doc-devel/lib64/python3.11/site-packages/staticjinja/staticjinja.py", line 329, in get_rule
    raise ValueError("no matching rule")
ValueError: no matching rule

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/samccann/jinja-docsite/build.py", line 27, in <module>
    site.render(use_reloader=False)
  File "/home/samccann/venvs/jinja-doc-devel/lib64/python3.11/site-packages/staticjinja/staticjinja.py", line 475, in render
    self.render_templates(self.templates)
  File "/home/samccann/venvs/jinja-doc-devel/lib64/python3.11/site-packages/staticjinja/staticjinja.py", line 424, in render_templates
    self.render_template(template)
  File "/home/samccann/venvs/jinja-doc-devel/lib64/python3.11/site-packages/staticjinja/staticjinja.py", line 413, in render_template
    template.stream(**context).dump(filepath, self.encoding)
  File "/home/samccann/venvs/jinja-doc-devel/lib64/python3.11/site-packages/jinja2/environment.py", line 1618, in dump
    fp.writelines(iterable)
  File "/home/samccann/venvs/jinja-doc-devel/lib64/python3.11/site-packages/jinja2/environment.py", line 1613, in <genexpr>
    iterable = (x.encode(encoding, errors) for x in self)  # type: ignore
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samccann/venvs/jinja-doc-devel/lib64/python3.11/site-packages/jinja2/environment.py", line 1662, in __next__
    return self._next()  # type: ignore
           ^^^^^^^^^^^^
  File "/home/samccann/venvs/jinja-doc-devel/lib64/python3.11/site-packages/jinja2/environment.py", line 1354, in generate
    yield self.environment.handle_exception()
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samccann/venvs/jinja-doc-devel/lib64/python3.11/site-packages/jinja2/environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "/home/samccann/jinja-docsite/templates/community.html", line 1, in top-level template code
    {% extends "_base.html" %} {% block body %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samccann/jinja-docsite/templates/_base.html", line 22, in top-level template code
    {% block body %} {% endblock %}
^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/samccann/jinja-docsite/templates/community.html", line 16, in block 'body'
    <a class="btn" href="{{ pages.site.ecosystem }}" role="button">{{ pages.community.collaborate.ecosystem }}</a>
^^^^^^^^^^^^^^^^^^^
  File "/home/samccann/venvs/jinja-doc-devel/lib64/python3.11/site-packages/jinja2/environment.py", line 485, in getattr
    return getattr(obj, attribute)
           ^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'site'`

**Review by samccann — COMMENTED** [review]
I can't see the html because a local build doesn't work for me on this PR.

From what I can groc in the source, you fixed the nits I had in my last review..

**Comment by oraNod** [comment]
@samccann Try again. One of the other PRs added a link that this PR changes...

**Review by samccann — APPROVED** [review]
LGTM

**Comment by oraNod** [comment]
thanks for all the help @samccann

**Pull request merged**